app/models/customization.rb

```diff
@@ -0,0 +1,11 @@
+# typed: strict
+class Customization < ApplicationRecord
+  belongs_to :item
+
+  # Define the attributes with their respective data types and constraints
+  attribute :id, :integer
+  attribute :created_at, :datetime
+  attribute :updated_at, :datetime
+  attribute :customization_details, :text
+  attribute :item_id, :integer
+end
```

Comment:
The Customization model directly defines attributes with data types, which is not necessary as the schema is already defined in the migrations. This could lead to inconsistencies between the model and the database schema. Suggest removing the attribute definitions and relying on the schema provided by the migrations.